examples/vpc_peering_common_dual_stack/main_test.go

```diff
@@ -0,0 +1,65 @@
+package vpc_peering_common
```

Comment:
let's maybe call this vpc_peering_common_ipv6

Reply:
Yes, I should have chosen better name